src/components/LoginForm.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
```

Comment:
The cocogoose font is actually in the main branch and not in the dev for some reason, so if you want to change the font to the figma then the file is in public/fonts/cocogoose.tff

Reply:
I'll choose not to include the font for the current PR since I assume it'll be implemented into the typography once theming is added.